_styles/main.css

```diff
@@ -97,6 +82,10 @@ h1 .logotype-svg {
     }
 }
 
+h1 + p {
+    font-size: 1.25em;
+}
```

Comment:
I'm not sure I'm a fan of this. Maybe something a little more subtle like:
```suggestion
h1 + p {
    font-size: 1.15em;
}
```

Reply:
Alternatively (or maybe additionally) I feel like we should decrease line height a bit here. As in:

![Screenshot from 2020-09-17 10 29 42@2x](https://user-images.githubusercontent.com/7277719/93506206-cb8b2000-f8d0-11ea-8fa5-aa0e81a54a39.png)